src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-671.mdx

```diff
@@ -0,0 +1,20 @@
+---
+subject: Android agent
+releaseDate: '2022-07-27'
+version: 6.7.1
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_6.7.1.zip'
+---
+
+## New in this release
+* Add support for handling JavaScript Errors from React Native apps
+* Add harvest data usage supportibility metrics
+* Add new API to detect whether the hosting device is rooted
+
+
+## Fixed in this release
+* Fixed an issue with distributed tracing header
+* Fixed an issue to address plugin concurrency
```

Comment:
We didn't really. What that change did was address a corner-case exception thrown during map processing.

Reply:
I modified the language in the notes